docs/docs/forms.md

```diff
@@ -76,6 +76,8 @@ handleChange(event) {
 }
 ```
 
+However, if `this.state.value` becomes `null` or `undefined` at any time (including the initial state), React will evaluate the component as uncontrolled and may produce a warning: `Input elements should not switch from uncontrolled to controlled (or vice versa)`.
+
```

Comment:
@Aprillion I do get that, and it makes sense from that perspective, but these docs are primarily accessed as a primary source for users learning/using React. I think including the warning might serve to confuse the majority of users who aren't coming from the link in the warning.

We could potentially make the warning itself clearer if there's some specific ambiguity you think we could clarify.